app/views/applications/_show.html.haml

```diff
@@ -8,9 +8,16 @@
 
   %tr
     %td.left
-      Email
+      Contact email
       %small (required)
+      %p.small *We will contact you using this email, and we will add this email to the DU mailing list if you get accepted as a member.
```

Comment:
Just checking -- which email do we add to the Google group, the contact email or the Google-friendly email?

I assumed in this PR that we add their contact email to the mailing list, since we require everyone to give a contact email but we don't require everyone to give a Google-friendly email. It looks like non-Google emails can be in Google groups. 

However, I just noticed that the setup page for members who are accepted to DU says that we use their Google-friendly email address to add them to the mailing list:
![Screenshot 2022-07-19 10 33 30 PM](https://user-images.githubusercontent.com/5607966/179904694-15e12e9f-2fe7-4659-87d1-b9c2aebd2b4b.png)

I can either update this text or update that page depending on which one is right.

Reply:
@brittag do you know what is our intent, and what should this wording be?

Reply:
It is possible that currently we also use the members mailing list to share the Google Drive with all members in an easy way, although that doesn't have to be necessarily the case. If we want to share Google docs with the mailing list, then I think that implies that we want the mailing list emails to be the "email for google" fields, which might not be what members expect. 🤔 

(I don't love the whole "two emails" situation, which I believe exists only so that we can share the Google Drive folder with folks. It sure would be nice to find a different solution, but that's for another time.)

Reply:
I just checked the Members Google Calendar, and indeed it is shared with the members group. We could plan to at least create a separate, no-email group with google email addresses for calendar and document sharing, and we could even eventually write code to update that group automatically. But that's a whole different task.

So from a current functional perspective, we want to add "email for google" to the members mailing list, to maximize the chances that members have access to the calendar and documents.

Reply:
@anaulin @sarahwalters Oh, this is a great question. I don't know more than you do about this! I'd say go forward with adding "email for google" to the mailing list.

Reply:
Thank you both! I updated the language to say that we'll use the contact email to contact people about their application, and we'll use the Google-friendly email to add them to the Google group / drive / calendar if they get accepted to DU.

Agreed that it's confusing to ask for two emails, but consolidating on one email is a different task.